src/kirby/components/calendar/calendar.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="header">
+  <button [disabled]="!canNavigateBack" (click)="changeMonth(-1)">
+    <kirby-icon name="arrow-back"></kirby-icon>
+  </button>
+
+  <div class="month-and-year"><span class="month">{{ activeMonthName }}</span><span class="year">{{ activeYear }}</span></div>
+
+  <button [disabled]="!canNavigateForward" (click)="changeMonth(1)">
+    <kirby-icon name="arrow-forward"></kirby-icon>
+  </button>
+</div>
+
+<table class="calendar-table">
+  <thead class="calendar-table-header">
+    <th *ngFor="let weekDay of weekDays">{{ weekDay }}</th>
+  </thead>
+    
+  <tbody>
+    <tr *ngFor="let week of month">
+      <td *ngFor="let day of week">
+        <div (click)="onDateSelected(day)" class="{{day.cssClasses}}" [class.selected]="day.isSelected">
+          {{ day.date }}
+        </div>
+      </td>
+    </tr>
+  </tbody>
+</table>
+  
+<!-- <iframe src="kirby/components/calendar/calendar.webview.html" #calendarContainer style="width: 320px; height: 304px; border: 0"> -->
```

Comment:
Consider removing this comment?

Reply:
I'll keep it for now as an easy way to test the webview implementation. It'll be transpiled away anyway...